codechecker_lib/analyzers/result_handler_plist_to_file.py

```diff
@@ -0,0 +1,82 @@
+# -------------------------------------------------------------------------
+#                     The CodeChecker Infrastructure
+#   This file is distributed under the University of Illinois Open Source
+#   License. See LICENSE.TXT for details.
+# -------------------------------------------------------------------------
+
+import ntpath
+import os
+import shutil
+from abc import ABCMeta
+
+from codechecker_lib import plist_parser
+from codechecker_lib.logger import LoggerFactory
+from codechecker_lib.analyzers.result_handler_base import ResultHandler
+
+LOG = LoggerFactory.get_new_logger('PLIST TO FILE')
+
+
+class PlistToFile(ResultHandler):
+    """
+    Result handler for copying a plist file to a different location.
+    """
+
+    __metaclass__ = ABCMeta
+
+    def __init__(self, buildaction, workspace, lock, export_plist_path):
+        super(PlistToFile, self).__init__(buildaction, workspace)
+        self.__lock = lock
```

Comment:
There is a uniuque plist filename created for the SA,
see: https://github.com/Ericsson/codechecker/blob/master/codechecker_lib/analyzers/result_handler_base.py#L171

I did not find suddenly what happens to clang tidy's output (which is passed to PListConverter). But in case that is unique as well, we should not do any locking. The locking is there for the standard output.